frontend/src/about.js

```diff
@@ -9,29 +9,30 @@ export default () => (
     </h3>
     <p>
       This tool uses a machine learning model trained on 1.7 million{' '}
-      <a href='https://www.ncbi.nlm.nih.gov/pmc/'>PubMed Central</a> documents
-      to recommend suitable journals based on the textual content of your{' '}
-      <a href='https://www.biorxiv.org/'>bioRxiv</a> preprint. This tool
-      provides a "map" of all documents within{' '}
       <a href='https://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/'>
-        {' '}
-        PubMed Central's Open Access Corpus{' '}
-      </a>
-      , and plots where your <a href='https://www.biorxiv.org/'>bioRxiv</a>{' '}
-      preprint falls on this map. Each square bin on the map represents a group
-      of papers that share similar textual content. Click on a square to view
-      more information about papers contained in a given bin.
+        PubMed Central open access documents
+      </a>{' '}
+      to recommend suitable journals based on the textual content of your{' '}
+      <a href='https://www.biorxiv.org/'>bioRxiv</a> preprint.
+    </p>
+    <p>
+      The tool also provides a "map" of the documents, grouped into bins based
+      on similar textual content, and shows you where your preprint falls on it.
+      Select a square to learn more about the papers in that bin.
+    </p>
+    <p>
+      The map also incorporates a set of 50{' '}
+      <a href='https://en.wikipedia.org/wiki/Principal_component_analysis'>
+        principal components
+      </a>{' '}
+      (PCs) generated from the documents. Each PC represents two high level
```

Comment:
```suggestion
      (PCs) generated from the bioRxiv repository. Each PC represents two high level
```

Reply:
I thought the PC's were generated from the same 1.7 million PMC documents? If we're talking about all of bioRxiv in general, can we just say "generated from bioRxiv." ?

Reply:
Trying to keep this all as high level as possible, deferring details to the github readme or the full papers you wrote about it.

Reply:
Yeah you can say generated from bioRxiv. The PCs themselves were generated from bioRxiv and just overlaid on top of PubMed Central.